### PR TITLE
:bug: fix swap fetching balances

### PR DESCRIPTION
**Pull request by git-marcopitra**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206565322854037

### CODE REVIEW

File: pages/swap.tsx [hunk 1]
```diff
@@ -1,74 +1,110 @@
...
     },
   });
 
-  useEffect(() => {
+  const setDefaultToken = async (value: string, field: 'to' | 'from') => {
+    if (value === SUI_TYPE_ARG) {
+      const { type, symbol, decimals } =
+        COIN_TYPE_TO_COIN[network][SUI_TYPE_ARG];
+
+      const usdPrice = await fetch(`/api/v1/coin-price?symbol=${symbol}`)
```
Comment (MarioBatalha) [inline pages/swap.tsx:43]: USD price is not displayed

File: views/swap/input/dollar-value.tsx [hunk 2]
```diff
@@ -28,12 +28,8 @@ const AmountInDollar: FC<InputProps> = ({ label }) => {
       variant="body"
       color={value ? 'onSurface' : 'outline'}
     >
-      {usdPrice
-        ? formatDollars(
-            BigNumber(value || 0)
-              .times(BigNumber(usdPrice))
-              .toNumber()
-          )
+      {usdPrice && value
```
Comment (MarioBatalha) [inline views/swap/input/dollar-value.tsx:31]: For some reason, it's not working properly

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

File: views/swap/input/dollar-value.tsx (hunk 2 above)
Comment (git-marcopitra) [reply views/swap/input/dollar-value.tsx:31]: what you mean with display, can you send a screenshot?

Comment (MarioBatalha) [reply views/swap/input/dollar-value.tsx:31]: Screenshot 2024-02-13 at 19 27 14


USD price is not displayed

**Review by MarioBatalha — APPROVED** [review]

**Pull request merged**